src/Service/Resolvers/CosmosClientProvider.cs

```diff
@@ -117,5 +127,11 @@ private void ParseCosmosConnectionString()
             _accountEndpoint = dbConnectionStringBuilder.ContainsKey("AccountEndpoint") ? (string)dbConnectionStringBuilder["AccountEndpoint"] : null;
             _accountKey = dbConnectionStringBuilder.ContainsKey("AccountKey") ? (string)dbConnectionStringBuilder["AccountKey"] : null;
         }
+
+        private static string GetCosmosUserAgent()
+        {
+            return Environment.GetEnvironmentVariable(DAB_APP_NAME_ENV) ?? DEFAULT_APP_NAME;
```

Comment:
Is it possible to add the engine version too here? Can help understand patterns around adoption.

Reply:
Done